<launch>
    <arg name="pkg" value="rss_demo" />
    <arg name="type" default="joint_state_retargeter" />
    <arg name="name" default="$(arg type)" />
    <arg name="rate" default="30" />

    <!-- Node subscribes to /joint_state_retargeter/source_joint_states -->
    <!-- Node publishes to /joint_state_retargeter/target_joint_states -->

    <arg name="source_urdf" default="$(find rss_demo)/pr2_urdf/pr2.urdf" />
    <arg name="target_urdf" default="$(find bandit_urdf)/urdf/bandit_defs.urdf"/>
    
    <arg name="args" value="_loop_rate:=$(arg rate)
			    _source_urdf:=$(arg source_urdf) 
			    _target_urdf:=$(arg target_urdf)" />

    <!-- Load the endpoints for the chains that we will be retargeting -->
    <rosparam param="/$(arg name)/chain_info" command="load" file="$(find rss_demo)/params/pr2_both_bandit_both.yaml" />

    <node
        pkg="$(arg pkg)"
        type="$(arg type)_node"
        name="$(arg name)"
        args="$(arg args)"
        output="screen" />
</launch>
